stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/querychannels/QueryChannelsController.kt

```diff
@@ -120,9 +109,8 @@ public class QueryChannelsController internal constructor(
     }
 
     internal suspend fun updateQueryChannelSpec(channel: Channel) {
-        if (newChannelEventFilter(channel, filter)) {
+        if (defaultChannelEventsHandler.newChannelEventFilter(channel, filter)) {
```

Comment:
```suggestion
        if (eventsHandler.newChannelEventFilter(channel, filter)) {
```

Reply:
It's not right. This old channel event filter is only kept now in `DefaultChannelEventsHandler `.  Maybe we should delete this method completely

Reply:
Not exactly. It is used in ChatDomain here (I believe we can use here `addChannel`):
![image](https://user-images.githubusercontent.com/11807573/136045806-7e29d18c-41c3-4756-8c87-d90c863dfac7.png)

And we use old event filter here (IDK what we have to do here):
![image](https://user-images.githubusercontent.com/11807573/136046238-04cb2e14-dde9-43f5-bbd5-0b26868ab2e5.png)